src/System.Console/src/project.json

```diff
@@ -26,6 +26,27 @@
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
       }
+    },
+    "netcore50": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.0.0-rc4-24126-00",
+        "Microsoft.TargetingPack.Private.WinRT": "1.0.1",
+        "Microsoft.NETCore.Platforms": "1.0.1-rc4-24126-00",
+        "System.Collections": "4.0.0",
+        "System.Diagnostics.Contracts": "4.0.0",
+        "System.Diagnostics.Debug": "4.0.10",
+        "System.Diagnostics.Tools": "4.0.0",
+        "System.IO": "4.0.10",
+        "System.IO.FileSystem.Primitives": "4.0.0",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.0.10",
+        "System.Runtime.InteropServices": "4.0.20",
+        "System.Text.Encoding": "4.0.10",
+        "System.Text.Encoding.Extensions": "4.0.10",
+        "System.Threading": "4.0.10",
```

Comment:
Can you have a look at the assembly references from the built assembly and reduce this list to only contain the assembly references from the final output?